src/components/FieldInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Field } from "react-final-form";
+
+const inputList = ["Email", "Password"];
+
+const TextInput = ({ nameform }: { nameform: string }) => {
```

Comment:
@bbun0639 I suggest you to use   https://final-form.org/docs/react-final-form/api/useField (hooks) instead of component because it's more concise

Reply:
also I suggest to separate label text and name of field because if you try to use same, it can lead to bug easily by just change label. so I think it's not worth.